viewer/src/utilities/metrics.ts

```diff
@@ -4,7 +4,7 @@
 
 import mixpanel from 'mixpanel-browser';
 
-type TrackedEvents = 'init' | 'loadModel' | 'error';
+type TrackedEvents = 'init' | 'construct3dViewer' | 'loadModel' | 'error' | 'navigated';
```

Comment:
Sorry if I just missed it (reviewing from phone) but how init differs from construct and why we need navigate ? What does it do at all? Navigate feels like app level metric if it’s about routes. 🤷‍♂️

Reply:
Aha, navigation is session update. A bit tricky :)

Reply:
do we have a better word?